src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -160,7 +180,7 @@ public string Title
         /// <summary>
         /// Title content of AlertDialog. TitleContent is added to Children automatically.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
"TitleContent is added as a child of AlertDialog automatically. " is better?

Reply:
Thank you for your comment.
I think your description is better for understanding.
The description has been updated.